e2e_tests/integration/multi-db.spec.js

```diff
@@ -38,6 +38,27 @@ describe('Multi database', () => {
     cy.connect('neo4j', password)
   })
   if (Cypress.config('serverVersion') >= 4.0) {
+    it('shows a message indicating whether system updates have occured', () => {
+      cy.executeCommand(':clear')
+
+      cy.executeCommand(':use system')
+      cy.executeCommand('CREATE DATABASE test1')
+
+      cy.wait(3000) // CREATE database can take a sec
+
+      cy.resultContains('1 system update, no records')
+
+      cy.executeCommand('STOP DATABASE test1')
```

Comment:
I know this isn't very pretty - doing `STOP DATABASE` twice in order to get 0 system updates. So suggestions welcome. 

I also haven't figured out a way to get >1 system updates to test pluralisation...

Reply:
Maybe I can add a unit test for the `/Stream/CypherFrame/helpers.js` instead for this. I will try this first and then return.

Reply:
No, I think this is a good approach 👍